plugins/DynamicsProcessor/DynamicsProcessor.cpp

```diff
@@ -75,8 +79,6 @@ DynProcEffect::DynProcEffect( Model * _parent,
 
 DynProcEffect::~DynProcEffect()
```

Comment:
You can get rid of the user defined destructor as well since its empty.

Reply:
Can you please clarify what you mean? Do you want me to delete the destructor only in `DynamicsProcessor.cpp`?

Reply:
You can remove both the declaration of `~DynProcEffect` in `DynamicsProcessor.h` and its definition (the one here) in `DynamicsProcessor.cpp` since its empty.